src/components/Shared/Forms/Input.tsx

```diff
@@ -0,0 +1,56 @@
+import { useTranslations } from 'next-intl';
+
+type InputProps = {
+  id: string;
+  label: string;
+  type: 'text' | 'email' | 'password' | 'number';
+  name: string;
+  placeholder: string;
+  errors?: string[] | null;
+  children?: React.ReactNode;
+  register?: Record<string, unknown>;
+};
+
+export default function Input({
+  id,
+  type,
+  name,
+  label,
+  errors,
+  children,
+  placeholder,
+  register = {},
+}: InputProps) {
+  const t = useTranslations('Forms');
+
+  return (
+    <div className="form-group">
+      <div className="form-row">
+        <label className="form-label" htmlFor={id}>
+          {label}:
+        </label>
+        <div className="input-wrapper">
+          <input
+            className="form-input"
+            type={type}
+            id={id}
+            name={name}
+            defaultValue=""
+            placeholder={placeholder}
```

Comment:
autoComplete is missing https://developer.mozilla.org/en-US/docs/Web/HTML/Reference/Attributes/autocomplete